src/ScriptCs.Core/App_Packages/LibLog.3.1/LibLog.cs

```diff
@@ -1814,7 +1814,7 @@ internal static class LogMessageFormatter
         /// <summary>
         /// Some logging frameworks support structured logging, such as serilog. This will allow you to add names to structured data in a format string:
         /// For example: Log("Log message to {user}", user). This only works with serilog, but as the user of LibLog, you don't know if serilog is actually 
-        /// used. So, this class simulates that. it will replace any text in {curlybraces} with an index number. 
+        /// used. So, this class simulates that. it will replace any text in {curly braces} with an index number. 
         /// 
         /// "Log {message} to {user}" would turn into => "Log {0} to {1}". Then the format parameters are handled using regular .net string.Format.
         /// </summary>
```

Comment:
This file is imported by the LibLog package so we really shouldn't edit it. Any changes here should be submitted as a PR to https://github.com/damianh/LibLog